scripts/build-worker.sh

```diff
@@ -54,6 +60,19 @@ pushd "$(dirname "$0")/../"
             +buildWorkerTypes "${WORKER_TYPE}" \
             +buildEnvironment "${BUILD_ENVIRONMENT}" \
             +scriptingBackend "${SCRIPTING_BACKEND}" \
+            "${TARGET_IOS_SDK_ARG}" \
             "${BUILD_TARGET_FILTER_ARG}"
+
+        if [[ "${BUILD_TARGET_FILTER-}" =~ "ios" ]]; then
+            echo "--- Building XCode Project :hammer_and_wrench:"
+
+            dotnet run -p "${RUN_UNITY_PATH}" -- \
+                -projectPath "." \
+                -batchmode \
+                -quit \
+                -logfile "$(pwd)/../logs/${WORKER_TYPE}-xcode-build.log" \
```

Comment:
```suggestion
                -logfile "$(pwd)/../logs/${WORKER_TYPE}-${BUILD_ENVIRONMENT}-xcode-build.log" \
```

Prevent contention between local and cloud :)